source/Halibut.Tests/Support/TestAttributes/FailedWebSocketTestsBecomeInconclusiveAttribute.cs

```diff
@@ -1,70 +1,11 @@
 using System;
-using System.Linq;
-using Halibut.Tests.Support.TestCases;
 using NUnit.Framework;
-using NUnit.Framework.Interfaces;
-using NUnit.Framework.Internal;
-using NUnit.Framework.Internal.Commands;
 
 namespace Halibut.Tests.Support.TestAttributes
 {
     [AttributeUsage(AttributeTargets.Method)]
-    public class FailedWebSocketTestsBecomeInconclusiveAttribute : PropertyAttribute, IWrapSetUpTearDown, IApplyToContext
+    [Obsolete("This is a noop attribute now")]
+    public class FailedWebSocketTestsBecomeInconclusiveAttribute : PropertyAttribute
```

Comment:
This is because websockets are now finally reliable?

Reply:
Yes, I didn't want to remove the attribute and make loads of noise in this PR, but I wanted to make sure WebSocket failures failed the build